src/main/resources/com/cognifide/gradle/aem/environment/README.md

```diff
@@ -0,0 +1,94 @@
+
+# Local AEM Environment Setup with Docker
+
+This setup consists of a dispatcher connected by default with AEM publish and AEM author deployed via [gradle-aem-plugin](https://github.com/Cognifide/gradle-aem-plugin). Dispatcher runs on docker on a [official httpd image](https://hub.docker.com/_/httpd).
+
+AEM author and publish instances need to be up.
+
+## Environment in details
+1. AEM author at [http://localhost:4502](http://localhost:4502)
```

Comment:
I think that after moving these domains, `example.com` and so on to gradle-aem-multi, in docs here we will could just point to link to gradle-aem-multi to section describing flow of working with automated instance/env (instead of describing gradle-aem-multi in GAP / I want to avoid that)